sdk/cosmosdb/cosmos/.gitignore

```diff
@@ -261,9 +261,14 @@ lib/**
 
 *.tgz
 
+dist-esm/
```

Comment:
In general I tried to standardize all git excludes in the top level `.gitignore` file for the repo, however if you have extra ones that are specific to cosmos I think it's fine to have a separate file here.

Reply:
For now it is needed because build output of cosmos tests are not yet into the directory in the monorepo convention.